x/authenticator/ante/circuit_breaker.go

```diff
@@ -70,5 +80,13 @@ func IsCircuitBreakActive(
 		return true, nil
 	}
 
+	// Retrieve the selected authenticators from the extension.
+	selectedAuthenticators := txOptions.GetSelectedAuthenticators()
+
+	if len(selectedAuthenticators) < 1 {
+		// Return true if the number of selected authenticators is less than 1.
+		return true, nil
+	}
+
```

Comment:
Code quality: this looks like AI generated code with a lot of useless pieces
```suggestion
	if len(txOptions.GetSelectedAuthenticators()) < 1 {
		return true, nil
	}
```

maybe even mergeable with previous if statement

```golang
if txOptions == nil || len(txOptions.GetSelectedAuthenticators()) < 1 {
	return true, nil
}
```

Reply:
Thanks for your feedback! 

I've updated the statement to inline the check you highlighted:
https://github.com/osmosis-labs/osmosis/pull/7683/commits/a22be6e1f6f1a78dcd0bb81315bc3242f791b9cc